Indicators/MovingAverageConvergenceDivergence.cs

```diff
@@ -36,6 +38,13 @@ public class MovingAverageConvergenceDivergence : Indicator
         /// </summary>
         public IndicatorBase<IndicatorDataPoint> Signal { get; private set; }
 
+        /// <summary>
+        /// Developed by Thomas Aspray in 1986, the MACD-Histogram measures the distance between MACD and its signal line, 
+        /// is an oscillator that fluctuates above and below the zero line.
+        /// Bullish or bearish divergences in the MACD-Histogram can alert chartists to an imminent signal line crossover in MACD.
+        /// </summary>
+        public decimal Histogram { get; private set; }
```

Comment:
Why is this not a `IndicatorBase<IndicatorDataPoint>` like the other variables?

Reply:
I tried using with a `CompositeIndicator` in the first commit, but it throws error when it is reset.

But I see your point, the code will more consistent, I'll make an implementation using and `Identity` indicator for the Histogram.